daemon/src/to_sse_event.rs

```diff
@@ -103,7 +103,10 @@ impl ToSseEvent for CfdsWithCurrentPrice {
             .cfds
             .iter()
             .map(|cfd| {
-                let (profit_btc, profit_usd) = cfd.profit(current_price).unwrap();
+                let (profit_btc, profit_in_percent) = cfd
+                    .profit(current_price)
+                    .map(|(p_l, in_percent)| (p_l.round_dp(6), in_percent.round_dp(2)))
```

Comment:
I'd prefer if we output nice domain types from our domain layer. (This is essentially the same feedback as above with not returning decimals from `profit`).

These domain types could then have more descriptive functions for what you are doing here. (Plus be tested in isolation).

I'd prefer the API layer to be rather stupid in regards to its mapping logic.

Reply:
What types do you want this domain type to wrap around? Is Decimal fine?

Reply:
Yeah it is fine if `Decimal` is internal type as long as it is an implementation detail!